ChangeLog

```diff
@@ -91,8 +95,7 @@
   ConsoleWriterFactory (#926).
   (contributed by @pljones)
 
-- Bug fix: A crash when settings are changed in ASIO4ALL during a connection
-  has been fixed (#796).
+- Bug fix: Changing ASIO driver properties like switching input/outputs in ASIO4ALL now resets input/output channel mapping in Jamulus again. This was done to solve a bug with ASIO4ALL introduced in the last version.
```

Comment:
The reference to #796 was dropped. Was that unintentional?

Reply:
I don't think it was intentional. I've restored it now. Thanks.

BTW: I also unified line breaks. We had some long entries and we had wrapped entries. I unified it so that the first sentence ("header") of each entry is always on a single line.